File: python/linear_search.py
def linear_search(value_to_find, array_to_search_through):
    found = None
    for i in range(len(array_to_search_through)):
        if value_to_find == array_to_search_through[i]:
            found = i
            
    return found

def linear_search_global(value_to_find, array_to_search_through):
    found = []
    for i in range(len(array_to_search_through)):
        if value_to_find == array_to_search_through[i]:
            found.append(i)
            
    return found

File: js/linearSearch.js
const linearSearch = function(valueToFind, arrayToSearchThrough){
   let found
   for(let i = 0; i < arrayToSearchThrough.length; i++){
      if(valueToFind === arrayToSearchThrough[i]){
         found = i
      }
   }
   return found
}

const linearSearchGlobal = function(valueToFind, arrayToSearchThrough){
   let found = []
   for(let i = 0; i < arrayToSearchThrough.length; i++){
      if(valueToFind === arrayToSearchThrough[i]){
         found.push(i)
      }
   }
   return found
}

module.exports.linearSearch = linearSearch
module.exports.linearSearchGlobal = linearSearchGlobal
